vcr/vcr.go

```diff
@@ -191,12 +191,22 @@ func (c *vcr) Search(query concept.Query) ([]did.VerifiableCredential, error) {
 	return VCs, nil
 }
 
-func (c *vcr) Issue(vc did.VerifiableCredential) (*did.VerifiableCredential, error) {
-	validator, builder := credential.FindValidatorAndBuilder(vc)
+func (c *vcr) Issue(template did.VerifiableCredential) (*did.VerifiableCredential, error) {
+	validator, builder := credential.FindValidatorAndBuilder(template)
 	if validator == nil || builder == nil {
 		return nil, errors.New("unknown credential type")
 	}
 
+	if len(template.Type) > 1 {
```

Comment:
`builder.fill()` does this all for you at line 223

Reply:
I don't agree: it adds only adds the `VerifiableCredentialType` if not present. We clearly state "only 1 type is supported" and thus we should validate it, since we chose to reuse the `did.VerifiableCredential` type where `type` is a slice and thus it allows passing more than 1 type.